plutus-use-cases/src/Language/Marlowe/Escrow.hs

```diff
@@ -0,0 +1,78 @@
+module Language.Marlowe.Escrow where
+
+import Language.Marlowe.Compiler
+import           Wallet.API                                          (PubKey (..))
+
+------------------------------------------
+-- Implementation of an escrow contract --
+------------------------------------------
+
+-- The contract allows person 1 to pay 450 ADA
+-- to person 2 by using person 3 as an escrow.
+--
+-- Person 1 can commit 450 ADA until block 100.
+-- The payment will go through only if 2 out of the 3
+-- participants choose the number "1".
+--
+-- If 2 out of the 3 participants chooses the number "0"
+-- or if payment did not go through after block 100,
+-- the money will be refunded to person 1.
+
+
+-- escrowContract :: Contract
+-- escrowContract = CommitCash 1
+--                     (Value 450)
+--                     (When (OrObs (two_chose 1 2 3 refund)
+--                                  (two_chose 1 2 3 pay))
+--                           (Choice (two_chose 1 2 3 pay)
+--                                   (Pay 1 2 Committed)
+--                                   redeem_original))
+
+-- refund = 0
+-- pay    = 1
```

Comment:
commented-out code

Reply:
This one I just copy/pasted from Marlowe repo. Technically it's a kind of documentation, to help understand what are those 0/1s mean in the actual contract below.